far/FarEng.hlf.m4

```diff
@@ -3,43 +3,44 @@
 .Options CtrlColorChar=\
 .Options CtrlStartPosChar=^<wrap>
 
+@CodepagesSettings=CodepagesMenu
 @ConfirmationsSettings=ConfirmDlg
-@InterfaceSettings=InterfSettings
-@ScreenSettings=InterfSettings
 @DescriptionsSettings=FileDiz
-@CodepagesSettings=CodepagesMenu
+@Editor.WordDiv=System.WordDiv
+@History.CommandHistory.Count=History.Config
+@History.CommandHistory.Lifetime=History.Config
+@History.DialogHistory.Count=History.Config
+@History.DialogHistory.Lifetime=History.Config
+@History.FolderHistory.Count=History.Config
+@History.FolderHistory.Lifetime=History.Config
+@History.ViewEditHistory.Count=History.Config
+@History.ViewEditHistory.Lifetime=History.Config
 @Interface.CompletionSettings=AutoCompleteSettings
-@Panel.Left=PanelCmdSort
-@Panel.Right=PanelCmdSort
-@Interface.ViewerTitleFormat=TitleFormat
-@Interface.EditorTitleFormat=TitleFormat
 @Interface.CursorSize1=Interface.CursorSize
 @Interface.CursorSize2=Interface.CursorSize
 @Interface.CursorSize3=Interface.CursorSize
 @Interface.CursorSize4=Interface.CursorSize
 @Interface.DelShowSelected=Interface.DelHighlightSelected
-@System.MsWheelDeltaView=System.MsWheelDelta
-@System.MsWheelDeltaEdit=System.MsWheelDelta
-@System.MsWheelDeltaHelp=System.MsWheelDelta
+@Interface.EditorTitleFormat=TitleFormat
+@Interface.ViewerTitleFormat=TitleFormat
+@InterfaceSettings=InterfSettings
+@Panel.InfoSettings=InfoPanel
+@Panel.Left=PanelCmdSort
+@Panel.Right=PanelCmdSort
+@Panel.Tree=TreeSettings
```

Comment:
This should be
`@Panel.TreeSettings=TreeSettings`
But then it resolves the `Panel.Tree.TreeFileAttr` `far:config` parameter to the same "Tree Settings" UI page, where there is no corresponding option exists.

Maybe define two separate aliases, as it was in the previous iteration?
```
Panel.Tree.AutoChangeFolder=TreeSettings
Panel.Tree.MinTreeCount=TreeSettings
```
What do you think? I am going to bed. Until tomorrow.

Reply:
Sure, that's fine.
My comment was about avoiding duplicates in the text if possible, not about minimising the number of aliases :)

Reply:
Thank you. Understood. Done.